### ISSUE TITLE
Multiple LABEL layers when one would do.

### ISSUE THREAD

**Issue by mateor** [CentOS/CentOS-Dockerfiles#58]
Most (all?) of the Dockerfiles in this repo define multiple `LABEL` layers. But instead of combining them into one layer, each LABEL has its own layer. 

      LABEL License=GPLv2
      LABEL Vendor=CentOS

instead of

      LABEL License=GPLv2, Vendor=CentOS

This creates two layers when one will do. This is generally considered an inefficient image and is [discouraged in the Docker docs](https://docs.docker.com/engine/userguide/labels-custom-metadata/). This appears to be a design decision but I couldn't find any supporting reasons.

**Comment by goern** [comment]
@mateor would you like to have a look at that or shall we simple not merge any PR that does not also fix what you mentioned? ;)

**Comment by mateor** [comment]
Sure, I can submit a patch. I wanted to ask first.

The reason I started looking at this was because of the Centos7 base - but I couldn't find those. Can you point me to them - I would like to patch those as well.

**Comment by goern** [comment]
sure, its over here> https://github.com/CentOS/sig-cloud-instance-images/blob/CentOS-7.2.1511/docker/Dockerfile

**Comment by mateor** [comment]
ah ha. Thanks. I will put together some patches.

**Comment by jperrin** [comment]
We've actually updated this in the ```latest``` and ```7``` tags. The https://github.com/CentOS/sig-cloud-instance-images/blob/CentOS-7/docker/Dockerfile has adjusted this. Would it meet your needs the way it's set up?

**Comment by mateor** [comment]
hey, look at that. Great! Looks fine to me, thanks for doing that.

**Comment by jperrin** [comment]
The downside is we don't update the minor point-release tags like 7.2.1511, or 6.7. The expectation with users/consumers is that those don't change or get updates. They're static images for better or worse.

**Comment by mateor** [comment]
Yes, I think that is a good and fair contract. This will make a difference to anyone consuming the new images though, so I am glad to see the change.

How does the merge process work for this repo? Should the pull request be per-subproject?
